src/r2mm/installing/profile_installers/ModLoaderVariantRecord.ts

```diff
@@ -76,5 +76,10 @@ const VARIANTS = {
 // Exported separately from the definition in order to preserve the key names in the type definition.
 // Otherwise this would become [key: string] and we couldn't use the game names for type hinting elsewhere.
 // Casting is done here to ensure the values are ModLoaderPackageMapping[]
+
+export const VARIANT_DEPENDENCIES: {[key: string]: string[]} = {
+    SkulTheHeroSlayer: ["UnityLibs-SkulTheHeroSlayer_UnityUnstripped"]
+}
```

Comment:
Rather than adding a new map here, could we just extend the `ModLoaderPackageMapping` class to include a field for the dependencies? That'd be a bit easier to follow in terms of the code & would probably be more semantically correct, as it's not the _game_ that's bound to those dependencies, but rather a specific mod loader package for that game.